explore/src/main/scala/explore/RootComponent.scala

```diff
@@ -26,34 +26,20 @@ case class RootComponent(
   ctx:          AppContext[IO],
   router:       RouterWithProps[Page, RootModelViews],
   initialModel: RootModel
-) extends ReactFnProps(RootComponent.component)
+) extends ReactFnProps(RootComponent)
 
-object RootComponent:
-  private type Props = RootComponent
-
-  val instrumentations =
-    js.Array(getWebAutoInstrumentations(), new UserInteractionInstrumentation())
-
-  private val component =
-    ScalaFnComponent
-      .withHooks[Props]
-      .useStateViewBy(_.initialModel)
-      .useThrottlingStateView(pending[ProgramSummaries], 5.seconds)
-      .render: (props, rootModel, programSummariesPot) =>
-        AppContext.ctx.provide(props.ctx):
-          React.Fragment(
-            props.ctx.tracing.map: c =>
-              val attr = rootModel.get.vault.map(ResourceAttributes.fromUserVault)
-              Observability(
-                HoneycombOptions(
-                  c.key,
-                  c.serviceName,
-                  instrumentations = instrumentations,
```

Comment:
Why was instrumentations removed?

Reply:
It wasn't used. It had an `@unused` annotation in the called site. Do we plan to use it in the future?

Reply:
Right, I'm trying to remember why we had it